facebook/__init__.py

```diff
@@ -122,6 +122,29 @@ def get_objects(self, ids, **args):
         args["ids"] = ",".join(ids)
         return self.request(self.version + "/", args)
 
+    def search_objects_by(self, type, **args):
+        """Fetches all objects of a given type from the graph.
+
+        Valid types are: event, group, page, place, placetopic, and user
+        Most types require the argument q, except:
+        - place requires q, addres or center
+        - placetopic doesn't require any additional argument
+
+        https://developers.facebook.com/docs/graph-api/using-graph-api
+        """
+
+        if type not in ('page',
```

Comment:
I think having each type on its own line makes this hard to read. All of them should fit on the same line without going over 80 characters.

Also, since the list of types is used twice (here and in the error message below), I think that this list would be an excellent candidate to be a constant like `VALID_API_VERSIONS`.